data/filters/youtube-upcoming-videos.yaml

```diff
@@ -2,17 +2,15 @@ title: Hide upcoming Youtube videos
 tags:
   - youtube
 template: |
-  www.youtube.com##ytd-browse ytd-grid-video-renderer:has(ytd-thumbnail-overlay-time-status-renderer[overlay-style="UPCOMING"])
```

Comment:
Looking at [the history for that file](https://github.com/letsblockit/letsblockit/blame/main/data/filters/youtube-upcoming-videos.yaml), I don't think it's a old-old / old-new situation here:
- `ytd-rich-item-renderer` is used in the homepage 
- `ytd-grid-video-renderer` is still used in [the subscriptions page](https://www.youtube.com/feed/subscriptions)

Looks like we still need the three rules on that template.

Reply:
I think you're right.

Reply:
Put `ytd-rich-item-renderer` first because homepage is the first page.
Or should we keep the alphabetical order?

Reply:
Because the `git blame` info is so useful, I prefer not to add more diff than necessary. I started heavily commenting the [youtube-shorts.yaml](https://github.com/letsblockit/letsblockit/blob/main/data/filters/youtube-shorts.yaml) template, to make it easier to read in the future. WDYT about adding comments there to know what these rules match?

Reply:
I agree on letting things simple. Adding comments sound not very usefull in this case. It was only about a logical order.